yarn.lock

```diff
@@ -5059,6 +5059,36 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@metamask/chain-agnostic-permission@npm:^0.2.0":
```

Comment:
Ideally we could dedupe these versions

Reply:
would require another release of `@metamask/multichain-api-middleware`

Reply:
Tracking here https://github.com/MetaMask/MetaMask-planning/issues/4599